setup.cfg

```diff
@@ -43,7 +43,7 @@ zip_safe = true
 
 [options.extras_require]
 numpy = 
-	numpy>=1.1
+	numpy>=2.0
```

Comment:
I believe the changes you made are compatible with numpy 1.1 so this is not necessary, is it?

Reply:
You're right. I've reverted this change.